content/articles/user-input-and-while-loops-in-python/index.md

```diff
@@ -0,0 +1,198 @@
+
+### Prerequisites
+1. Python installed on your computer.
+2. Basic Python knowledge.
+
+### How the input() function works
+The `input()` function halts your program and waits the user to key in some data. When Python receives the user's input, store it in the variable that you choose to work with.
+
+For example, let's create a program that accepts a user's name and prints back the name.
+
+```python
+name = input("Please enter your name: ")
+print(name)
+#output
+#Please enter your name: John
+#John
+```
+
+The `input()` function requires one argument, that is, the instruction you want the user to see. In this example, Python executes the first line and requests the user to enter his name. The program will halt and wait for the user to enter his name and continue after he presses **ENTER**. The name of the user is then loaded in the variable *name*, then `print(name)` displays the user's name back.
+
+### Using int() to accept numerical input
+Any text the user enters using the `input()` function Python interprets it as a string. If your work is to print the input, the `input()` function works very well. But if you use the input as a number, an error arises. Consider the example below:
+
+```python
+amount = input("Please enter your amount? ")
+rate=0.056
+interest= amount*rate
+print("Your interest is : ")
+print(interest)
+```
+
+This example gives an error because Python interprets the input *amount* as a string. Python can't multiply a string and a float ie rate.
+
+To solve this problem we use the `int()` function which informs Python to use the input as a numeric value. The `int()` function converts the string form of a number to numerical form, shown below:
+
+```python
+amount = input("Please enter your amount? ")
+amount=int(amount)
+rate=0.056
+interest= amount*rate
+print("Your interest is : ")
+print( interest)
+```
+
+Output:
+```
+Please enter your amount? 9000
+Your interest is : 
+504.0
+```
+
+When we enter 9000 in this example, Python interprets it as a string. The amount is then converted to an integer using the `int()` function. Now Python calculates the value of the interest.
+
+### Introducing while loops
+In python, a *while* loop executes provided certain condition remains true.
+
+Syntax:
+```python
+while expression:
+ statement(s)
+```
+
+The following *while* loop counts from 10 to 15:
+
+```python
+count = 10
+while (count <=15):
+ print ('The count is:', count)
+ count = count + 1 #add 1 to count
+#output
+#The count is: 10
+#The count is: 11
+#The count is: 12
+#The count is: 13
+#The count is: 14
+#The count is: 15
+```
+
+The *while* loop continues to execute if the value of *count* is equal to or less than 15.
```

Comment:
Use back ticks for function names and keywords. Change "The *while* loop continues to execute if the value of *count* is equal to or less than 15." to "The `while` loop continues to execute if the value of `count` is equal to or less than 15."